pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -57,16 +53,12 @@
     __name__ = PYWASMCROSS_ARGS.pop("orig__name__")
 
 
-import dataclasses
-import re
-import shutil
 import subprocess
 from collections.abc import Iterable, Iterator
-from typing import Literal
+from typing import Literal, NamedTuple
```

Comment:
It always strikes me as pretty weird that `NamedTuple` is a concrete class that you subclass and it changes your behavior but it comes from the `typing` module.

Reply:
Me too. `NamedTuple` and `TypedDict` surprises always surprises me.